GlobalBehaviorandComponents/templates/managesamlidp.html

```diff
@@ -0,0 +1,72 @@
+{% extends templatename+"/template.html" %}
+
+{% block content %}
+       {% include '//components/template-modal.html' %}
+       <section class="bg-light py-5">
+            <div class="container">
+                <div class="d-flex align-items-center mb-5">
+                    <div class="icon-stack icon-stack-lg bg-primary text-white"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-users"><path d="M17 21v-2a4 4 0 0 0-4-4H5a4 4 0 0 0-4 4v2"></path><circle cx="9" cy="7" r="4"></circle><path d="M23 21v-2a4 4 0 0 0-3-3.87"></path><path d="M16 3.13a4 4 0 0 1 0 7.75"></path></svg></div>
+                    <div class="ml-3">
+                        <h2 class="mb-0">Configure SSO</h2>
+                        <p class="lead mb-0">Connect your SAML2.0 Compatible Identity System!</p>
+                    </div>
+                </div>
+                <div class="card border-bottom">
+                    <div class="card-body">
+                        <div class="separator-2"></div>
+                          <form class="form-horizontal" action="/updatesamlidp" method="POST" enctype="multipart/form-data">
+                              <div class="form-group input-group">
+                             	  <div class="input-group-prepend">
+                            		  <span class="input-group-text"> <i class="fa fa-tag"></i> </span>
+                            		</div>
+                            	  <input name="idpName" class="form-control" placeholder="IDP Friendly Name" type="text" value="{{ idp_info['name'] if idp_info['name']|length > 1 else '' }}">
+                              </div>
+
+                              <div class="form-group input-group">
+                                <div class="input-group-prepend">
+                                  <span class="input-group-text"> <i class="fa fa-code"></i> </span>
+                                </div>
+                                <input name="idpMetadata" class="form-control" placeholder="IDP Metadata" type="file" value="">
+                              </div>
+
+                              <div>
+                                - OR -
```

Comment:
Dan Z and I talked about a nicer way to do an "either you upload a SAML metadata file, or you enter stuff in manually" form.